scripts/travisci_run.sh

```diff
@@ -15,7 +15,7 @@ export TERM=dumb
 # Go tests in Travis
 ./bin/buck test \
   --num-threads=3 \
-  --test-selectors '!GoBinaryIntegrationTest'
+  --test-selectors '!GoBinaryIntegrationTest,!GoTestIntegrationTest'
```

Comment:
Alternatively, we can just add a new file like `scripts/travisci_test_filters.txt` and have one test per line on there and pass `--test-selectors @scripts/travisci_test_filters.txt` here.